src/resources/shared/openshift/create_clusters_aws_openstack.md

```diff
@@ -0,0 +1,55 @@
+### Create and Deploy cluster-a
+
+In this step you will deploy **cluster-a** in **aws** (or any other public cloud can be used) using the default IP CIDR ranges:
```

Comment:
You are missing a step to setup the AWS profile. Please just reuse https://submariner.io/getting-started/quickstart/openshift/vsphere-aws/#create-and-deploy-cluster-b-on-aws